akka-stream/src/main/scala/akka/stream/impl/fusing/ActorGraphInterpreter.scala

```diff
@@ -564,5 +566,8 @@ private[stream] class ActorGraphInterpreter(_initial: GraphInterpreterShell) ext
       }
   }
 
-  override def postStop(): Unit = activeInterpreters.foreach(_.tryAbort(AbruptTerminationException(self)))
+  override def postStop(): Unit = {
+    activeInterpreters.foreach(_.tryAbort(AbruptTerminationException(self)))
```

Comment:
I have no clue why this commit is in here, this should not be in this PR—will fix.